BlockSettleHW/trezor/trezorDevice.cpp

```diff
@@ -449,16 +452,17 @@ void TrezorDevice::handleTxRequest(const MessageData& data)
    {
    case bitcoin::TxRequest_RequestType_TXINPUT:
    {
-      // Legacy inputs support
       if (!txRequest.details().tx_hash().empty()) {
-         auto tx = prevTx(txRequest);
-         auto txIn = tx.getTxInCopy(txRequest.details().request_index());
-
          auto input = txAck.mutable_tx()->add_inputs();
-         input->set_prev_hash(txIn.getOutPoint().getTxHash().copySwapEndian().toBinStr());
-         input->set_prev_index(txIn.getOutPoint().getTxOutIndex());
-         input->set_sequence(txIn.getSequence());
-         input->set_script_sig(txIn.getScript().toBinStr());
+
+         auto tx = prevTx(txRequest);
+         if (tx.isInitialized()) {
+            auto txIn = tx.getTxInCopy(txRequest.details().request_index());
```

Comment:
Why are invidual txins of supporting transactions passed on their own?

Reply:
Sorry, I don't understand the question. Trezor request input details one by one. When hash is that we need to looks in supportingTXs.